docker-compose.yml

```
@@ -0,0 +1,402 @@
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:7.4.0
+    container_name: antcamp-zookeeper
+    ports:
+      - "${ZOOKEEPER_CLIENT_PORT}:${ZOOKEEPER_CLIENT_PORT}"
+    environment:
+      ZOOKEEPER_SERVER_ID: 1
+      ZOOKEEPER_TICK_TIME: 2000
+      ZOOKEEPER_CLIENT_PORT: ${ZOOKEEPER_CLIENT_PORT}
+    healthcheck:
+      test: [ "CMD-SHELL", "echo srvr | nc -w 2 localhost 2181 | grep Mode || exit 1" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)